blog/blog_server/server.go

```diff
@@ -152,15 +152,15 @@ func (*server) UpdateBlog(ctx context.Context, req *blogpb.UpdateBlogRequest) (*
 
 func (*server) DeleteBlog(ctx context.Context, req *blogpb.DeleteBlogRequest) (*blogpb.DeleteBlogResponse, error) {
 	fmt.Println("Delete blog request")
-	oid, err := objectid.FromHex(req.GetBlogId())
+	oid, err := primitive.ObjectIDFromHex(req.GetBlogId())
 	if err != nil {
 		return nil, status.Errorf(
 			codes.InvalidArgument,
 			fmt.Sprintf("Cannot parse ID"),
 		)
 	}
 
-	filter := bson.NewDocument(bson.EC.ObjectID("_id", oid))
+	filter := bson.M{"_id": oid}
```

Comment:
why not filter := primitive.M{"_id": oid} instead? bson.M is from another package